learner_pathway_progress/admin.py

```diff
@@ -1,3 +1,25 @@
 """
 Admin interface for learner_pathway_progress.
 """
+from django.contrib import admin
+
+from learner_pathway_progress.models import LearnerPathwayMembership
+
+
+class LearnerPathwayMembershipAdmin(admin.ModelAdmin):
+    """
+    Admin for LearnerPathwayMembership Model.
+    """
+
+    raw_id_fields = ('user', )
+    list_display = ('user', 'learner_pathway_uuid', 'created')
+
+    class Meta:
+        """
+        Metadata for LearnerPathwayMembership model admin.
+        """
+
+        model = LearnerPathwayMembership
+
+
+admin.site.register(LearnerPathwayMembership, LearnerPathwayMembershipAdmin)
```

Comment:
I think it would be more readable to use `@admin.register(LearnerPathwayMembership)` decorator on the `ModelAdmin` class instead of this.
Example Usage: https://github.com/openedx/taxonomy-connector/blob/master/taxonomy/admin.py#L14